Assets/Scripts/PlayerScripts/Health.cs

```diff
@@ -0,0 +1,39 @@
+using UnityEngine;
+using UnityEngine.UI;
+
+public class Health : MonoBehaviour
+{
+    [SerializeField] private int _playerHealth = default;
+    [SerializeField] private int _healthNumber = default;
+    [SerializeField] private Image[] _lifeContainer = default;
+    [SerializeField] private Sprite _fullContainer = default;
+    [SerializeField] private Sprite _emptyContainer = default;
```

Comment:
Rename to something more descriptive, like `_fullContainerSprite`, the word `container` is typically used for transform parents.